ghost/core/core/server/web/api/endpoints/admin/routes.js

```diff
@@ -135,6 +135,7 @@ module.exports = function apiRoutes() {
     router.get('/stats/mrr', mw.authAdminApi, http(api.stats.mrr));
     router.get('/stats/subscriptions', mw.authAdminApi, http(api.stats.subscriptions));
     router.get('/stats/referrers/posts/:id', mw.authAdminApi, http(api.stats.postReferrers));
+    router.get('/stats/referrers/history', mw.authAdminApi, http(api.stats.referrersHistory));
```

Comment:
```suggestion
    router.get('/stats/referrers', mw.authAdminApi, http(api.stats.referrersHistory));
```
The mrr and member_count endpoints also return a history. And I think it is better to stay consistent here. Initially the `/stats/members_count` endpoint also was `/stats/members/count-history`. But on review at the time, that was changed to be shorter, [ref](https://ghost.slack.com/archives/C02G9E68C/p1648737467414789?thread_ts=1648644801.253699&cid=C02G9E68C)

Reply:
Ah cool, I had originally kept is as `/referrers` but changed it to be more explicit later, will change it back :)

Reply:
Updated